app/views/guests/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h3>Guest: <%= @guest.name %>
+<h2><br>Rooms: <% @guest.rooms %>
+<br>Suite: <%= @hotel.type %>
+<br>Rate: <%= @hotel.rating %>
+<br>Hotel: <%= @hotel.name %>
```

Comment:
In your assertions you are doing `@hotel.type` and `@hotel.rating` but a hotel only has a `name` and a `location`. You're also calling on `@guest.room` but there isn't currently a link to a guest and a room in your database setup. Again, this all stems from the database missing the join table in the setup.